desktop-widgets/filterwidget2.cpp

```diff
@@ -6,7 +6,10 @@
 
 #include <QDoubleSpinBox>
 
-FilterWidget2::FilterWidget2(QWidget* parent) : QWidget(parent), ignoreSignal(false)
+FilterWidget2::FilterWidget2(QWidget* parent) :
+	QWidget(parent),
+	ignoreSignal(false),
+	isActive(false)
```

Comment:
I wonder if it were more robust to add a  public member function to `MultiFilterSortModel`:
```
bool MultiFilterSortModel::isActive() const
{
        return filterData.validFilter;
}
```

and query with
```
MultiFilterSortModel::instance()->isActive();
```

Reply:
Yes, that sounds like a better approach...